README.md

```diff
@@ -1,6 +1,92 @@
 # WinForms TreeView Getting Started
 [TreeViewAdv]( https://www.syncfusion.com/winforms-ui-controls/treeview) displays a collection of data in hierarchical tree structure and the data in tree view can be expanded and collapsed. It offers many features like drag-and-drop, load on demand, context menus and data binding. The control comes with complete design time support.
```

Comment:
Add platform name here and remove the space

[WinForms TreeView](https://www.syncfusion.com/winforms-ui-controls/treeview) displays...........

Reply:
Resolved